### PR TITLE
Make linter happy

### PR DESCRIPTION
**Pull request by suneeta-mall**

### CODE REVIEW

File: ecr/sync.go [hunk 1]
```diff
@@ -168,7 +174,7 @@ func (s *ECRSyncer) doSync() error {
 			fmt.Sprintf("Failed to sync with ECR for tag %s", s.Config.Tag), "", "error",
 			time.Now().UTC(), s.Config.Tag, s.Config.accountID)
 		s.recorder.Event(s.pod, corev1.EventTypeWarning, "ECRSyncFailed", "More than one image with tag was found")
-		return errors.New(fmt.Sprintf("Bad state: More than one image was tagged with %s", s.Config.Tag))
+		return fmt.Errorf("Bad state: More than one image was tagged with %s", s.Config.Tag)
```
Comment (simoncochrane) [inline ecr/sync.go]: I thought the errors package had a version of `Errorf`?

Comment (suneeta-mall) [reply ecr/sync.go]: not the golang one https://golang.org/pkg/errors/#New

Comment (simoncochrane) [reply ecr/sync.go]: True, but we're using https://github.com/pkg/errors here.

Comment (suneeta-mall) [reply ecr/sync.go]: doh! I will fix it up

**Pull request merged**